modules/javafx.controls/src/main/java/javafx/scene/control/Skin.java

```diff
@@ -60,6 +73,14 @@
      */
     public Node getNode();
 
+    /**
+     * Called by {@link Skinnable#setSkin(Skin)} on a pristine control, or after the
+     * previous skin has been uninstalled via its {@link #dispose()} method.
+     * This method allows a Skin to register listeners, add child nodes, set
+     * required properties and/or event handlers.
```

Comment:
It might be good to add a sentence at the end saying: "The default implementation of this method does nothing".

Reply:
thank you.